workers/ohsome_quality_analyst/indicators/currentness/indicator.py

```diff
@@ -0,0 +1,183 @@
+import logging
+from io import StringIO
+from string import Template
+
+import dateutil.parser
+import geojson
+import matplotlib.pyplot as plt
+
+from ohsome_quality_analyst.base.indicator import BaseIndicator
+from ohsome_quality_analyst.ohsome import client as ohsome_client
+
+
+class Currentness(BaseIndicator):
+    """Time distribution of the last changes to elements over the entire time range."""
+
+    def __init__(
+        self,
+        layer_name: str,
+        feature: geojson.Feature,
+    ) -> None:
+        super().__init__(
+            layer_name=layer_name,
+            feature=feature,
+        )
+        self.threshold_yellow = 0.6
+        self.threshold_red = 0.2
+        self.element_count = None
+        self.contribution_sum = 0
+        self.contributions = {}
+        self.ratio = {}
+
+    async def preprocess(self) -> None:
+        latest_ohsome_stamp = await ohsome_client.get_latest_ohsome_timestamp()
+        start = "2008-01-01"
+        self.end = latest_ohsome_stamp.strftime("%Y-%m-%d")
+        time_range = "{0}/{1}/{2}".format(start, self.end, "P1Y")
+        curr_year_start = "{0}-01-01".format(latest_ohsome_stamp.year)
+        curr_year_range = "{0}/{1}".format(curr_year_start, self.end)
```

Comment:
Here a comment would be helpful. Why do we need all those different time ranges?
For me it was not clear at first that `time_range` does only cover a time range till 2020 and not 2021 (I guess because the period is one year and 2021 is still ongoing?).